spec/support/submission_test_setup.rb

```diff
@@ -25,7 +25,7 @@ def initialize(options = {})
     @repo_path = 'remote_repo'
     create_bare_repo(@repo_path)
 
-    @course = Course.create!(name: course_name, source_backend: 'git', source_url: @repo_path, organization: organization)
+    @course = Course.create!(name: course_name, title: course_name, source_backend: 'git', source_url: @repo_path, organization: organization)
```

Comment:
This change would require changes in several other test files, if they expect to click a link with course's name on it. We changed it so course's title is shown instead.